hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/CloseContainerEventHandler.java

```diff
@@ -99,6 +101,14 @@ public void onMessage(ContainerID containerID, EventPublisher publisher) {
     }
   }
 
+  private String getContainerToken(ContainerID containerID) {
+    StorageContainerManager scm = scmContext.getScm();
+    return scm != null
+        ? scm.getContainerTokenGenerator()
+            .generateEncodedToken(getClass().getSimpleName(), containerID)
```

Comment:
should we pass meaningful name here instead of the class name as user for the container token?

Reply:
Thanks @xiaoyuyao for the review.  Do you mean we should use the UGI that SCM is running with?

Reply:
Yes. I mean a class name can be confusing here as the close container command are sent by scm. There are other cases where scm admin cli can send the close container command to datanode, where should also use proper user from UGI.